tests/transports-integrations/run_all_tests.py

```
@@ -0,0 +1,330 @@
+        try:
+            # Run the test with pytest
+            cmd = [sys.executable, "-m", "pytest", str(test_file)]
+
+            # Add pytest flags for better output
+            if verbose:
+                cmd.extend(["-v", "-s"])  # verbose and don't capture output
+            else:
+                cmd.append("-q")  # quiet mode
+
+            if verbose:
+                result = subprocess.run(
```

Ruff (0.11.9)

`subprocess` call: check for execution of untrusted input

(S603)